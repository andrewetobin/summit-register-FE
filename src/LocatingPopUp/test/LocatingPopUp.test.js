import React from "react";
import ReactDOM from "react-dom";
import LocationPopUp from "..";
import { shallow } from "enzyme";

describe("LocatingPopUp", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(<LocationPopUp />);
    expect(wrapper).toMatchSnapshot();
  });

  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<LocationPopUp />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
});
