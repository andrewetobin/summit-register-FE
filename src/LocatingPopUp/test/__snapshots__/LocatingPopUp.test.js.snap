// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocatingPopUp should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LocatingPopUp />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3>
          Locating...
        </h3>,
        <img
          alt="loading"
          className="loading-gif"
          src="loading.gif"
        />,
      ],
      "className": "LocatingPopUp",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Locating...",
        },
        "ref": null,
        "rendered": "Locating...",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "loading",
          "className": "loading-gif",
          "src": "loading.gif",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Locating...
          </h3>,
          <img
            alt="loading"
            className="loading-gif"
            src="loading.gif"
          />,
        ],
        "className": "LocatingPopUp",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Locating...",
          },
          "ref": null,
          "rendered": "Locating...",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "loading",
            "className": "loading-gif",
            "src": "loading.gif",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
